networking/configuring-a-custom-pki.adoc

```diff
@@ -0,0 +1,57 @@
+[id="configuring-a-custom-pki"]
+= Configuring a custom PKI
+include::modules/common-attributes.adoc[]
+:context: configuring-a-custom-pki
+
+toc::[]
+
+Some platform components (i.e., web console) use Routes for communication and,
+therefore, must trust other components' certificates to interact with them. If
+you're using a custom public key infrastructure (PKI), you must configure it so
+its privately signed CA certificates are recognized across the cluster.
+
+You can leverage the Proxy API to add cluster-wide trusted CA certificates. You
+must do this either during installation *or* at runtime.
```

Comment:
Not necessary to emphasize "or".

Also, asterisks are only used for GUI items. If you do need to emphasize, should be italics.